deploy/helm/metacontroller/templates/statefulset.yaml

```diff
@@ -34,22 +34,25 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command:
-          - /usr/bin/metacontroller
+          - {{ .Values.command | default "/usr/bin/metacontroller" }}
           args:
-          - --zap-log-level={{ .Values.zapLogLevel | default 4 }}
-          - --discovery-interval={{ .Values.discoveryInterval | default "20s" }}
-          {{ if .Values.zap.devel -}}
-          - --zap-devel={{.Values.zap.devel}}
-          {{ end }}
-          {{- if .Values.zap.encoder -}}
-          - --zap-encoder={{.Values.zap.encoder}}
-          {{ end }}
-          {{- if .Values.zap.stacktraceLevel -}}
-          - --zap-stacktrace-level={{.Values.zap.stacktraceLevel}}
-          {{ end }}
-          {{- if .Values.cacheFlushInterval -}}
-          - --cache-flush-interval={{.Values.cacheFlushInterval}}
-          {{ end -}}
+          # TODO: discuss, these args will always be passed causing dlv to fail.
```

Comment:
TODO: discuss, these args will always be passed causing dlv to fail.
 I believe we should simpy use commandArgs and remove all these flags but this is a breaking change (leaning toward this option to simplify) or update the values such that they are optionally applied

Reply:
I updated this PR to consolidate all metacontroller cli args as `commandArgs`. This is a breaking change.